### PR TITLE
Hide block

### PR DESCRIPTION
**Pull request by Minglee01**
@incuna/js 

please review

thanks

### CODE REVIEW

File: src/sms-password/templates/user_management/sms-password/reset_form.html [hunk 1]
```diff
@@ -6,35 +6,36 @@
     <p translate>You've successfully changed your password.</p>
 </div>
 
-<div class="form-group" ng-class="{'has-error': errors.phone_number}">
+<div class="form-group" ng-class="{'has-error': errors.phone_number || errors.non_field_errors}">
     <label for="phone_number" ng-show="fields.phone_number.label" ng-bind="fields.phone_number.label"></label>
     <input type="text" ng-model="data.phone_number" class="form-control" id="phone_number" autofocus>
     <div class="help-block" ng-show="errors.phone_number" ng-bind="errors.phone_number"></div>
+    <div class="help-block" ng-if="errors.non_field_errors" ng-bind="errors.non_field_errors"></div>
```
Comment (henrahmagix) [inline src/sms-password/templates/user_management/sms-password/reset_form.html]: `non_field_errors` shouldn't be assigned to just the phone number. They should be separate from all fields. If a project needs a specific error that's in `non_field_errors` to be on the phone number, there should be a backend change to assign the error to the phone field or a frontend change to do what you've done here.

File: src/sms-password/templates/user_management/sms-password/reset_form.html [hunk 2]
```diff
@@ -6,35 +6,36 @@
...
 </div>
 
 <button type="submit" class="btn btn-default"  ng-click="resetRequest()" translate>Request code</button>
 
+<div ng-show="requested">
```
Comment (henrahmagix) [inline src/sms-password/templates/user_management/sms-password/reset_form.html]: Why are all the fields being hidden until `requested` is true? The commit messages don't explain this.

What about when `requested === false`? What's shown then? Do you have an empty form?

File: src/sms-password/templates/user_management/sms-password/reset_form.html (hunk 1 above)
Comment (Minglee01) [inline src/sms-password/templates/user_management/sms-password/reset_form.html]: This was already like this. I just moved the line up

Comment (henrahmagix) [inline src/sms-password/templates/user_management/sms-password/reset_form.html]: I can see in the diff that it was on it's own next to `errors.detail`, not in the phone number field.

Comment (Minglee01) [inline src/sms-password/templates/user_management/sms-password/reset_form.html]: @henrahmagix  check out the jira case. im not sure what else i can put here

File: src/sms-password/templates/user_management/sms-password/reset_form.html [hunk 3]
```diff
@@ -34,8 +33,8 @@
         <div class="help-block" ng-show="errors.new_password2" ng-bind="errors.new_password2"></div>
     </div>
 
-    <div class="help-block" ng-if="errors.detail" ng-bind="errors.detail"></div>
-
     <button type="submit" class="btn btn-default" ng-click="changePassword()" translate>Update password</button>
 </div>
 
+<div class="has-error" ng-if="errors.detail" ng-bind="errors.detail"></div>
+<div class="has-error" ng-if="errors.non_field_errors" ng-bind="errors.non_field_errors"></div>
```
Comment (henrahmagix) [inline src/sms-password/templates/user_management/sms-password/reset_form.html]: You don't have to move them out of the `ng-show="requested"` block, they can be in that block. At that point the diff will show just indentation change for all elements being indented into that block

**Comment by maciejpi** [comment]
@Minglee01 needs master merging

**Comment by henrahmagix** [comment]
And https://github.com/incuna/angular-user-management/pull/20#discussion-diff-59009868 needs addressing

**Comment by Minglee01** [comment]
I thought this was no longer being done.

**Comment by Minglee01** [comment]
Im closing this as im under the impression that its not needed anymore.

**Pull request closed without merging**